### ISSUE TITLE
Update composer.json

### ISSUE THREAD

**Issue by golgote** [pear/Log#19]
If you install the module with Composer 1, you get these deprecation notices:

```
Deprecation Notice: Class Log_observer_mail located in ./vendor/pear/log/examples/observer_mail.php does not comply with psr-0 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
Deprecation Notice: Class Log_error_log located in ./vendor/pear/log/Log/error_log.php does not comply with psr-0 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
```

These could be fixed by adding `exclude-from-classmap` in the composer.json file: 

```
"autoload": {
    "exclude-from-classmap": ["/examples/"]
}
```

**Comment by jparise** [comment]
@golgote Thanks! Would you open a Pull Request with your suggested change?

**Comment by golgote** [comment]
I'll do that ASAP 👍

**Comment by MikeyMJCO** [comment]
This should be fixed with the merging of #21

**Comment by williamdes** [comment]
This can be closed, fixed in https://github.com/pear/Log/pull/21/commits/3242f82d0264b59b12845b5f40429c2ee6c8c056
Released in 1.13.3
